techniques/failures/F113.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Failure of Success Criterion 1.2.5 due to not using available pauses in dialogue to provide audio descriptions of important visual content</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/></head><body>
+   <h1>Failure of Success Criterion 1.2.5 due to not using available pauses in dialogue to provide audio descriptions of important visual content</h1>
+   <section class="meta"><p class="id">ID: F113</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section>
+   <section id="applicability"><h2>When to Use</h2>
+      <p> Applies to all synchronized media. </p>
+   </section>
+   <section id="description"><h2>Description</h2>
+      <p>This describes a failure condition for all techniques involving audio descriptions. Audio descriptions can either be provided as part of the soundtrack's original narration or be added to audio during pauses in existing dialogue. If important actions, characters, scene changes, and on-screen text are only conveyed visually, appropriate pauses in dialogue must be used to provide this information as audio descriptions.</p>
```

Comment:
staying away from the sacred word "must".  adding "might" to account for some of the "what about when pauses are meaningful" discussion from today's call. IMO, this also helps align what's said here to square with the following note that says "not all pauses are usable..."

i know this says "appropriate pauses" - but since there is no definition of "appropriate"...  my thought is this also helps align with the note that kind of outlines what might be considered appropriate or not.

```suggestion
      <p>This describes a failure condition for all techniques involving audio descriptions. Audio descriptions can either be provided as part of the soundtrack's original narration or be added to audio during pauses in existing dialogue. If important actions, characters, scene changes, and on-screen text are only conveyed visually, appropriate pauses in dialogue might need to be used to provide this information as audio descriptions.</p>
```

Reply:
@scottaohara 
> staying away from the sacred word "must".

IMO, the context of the use of "must" here is appropriate for a failure technique. The scenario is:
1. important visual information is present
2. appropriate pauses in the dialogue exist

So, the idea is that it has already been determined that the pauses are appropriate. In such a case, I do not believe there is any way to provide meet 1.2.5 except by adding audio descriptions to narrate missing important visuals?

If you agree with this sentiment, but do not feel that is clear from this wording, I'll attempt a rewrite.
If you disagree with this sentiment, maybe you can say what level of containment we need to determine a failure.

Note that I have tried to use the phrase "appropriate pauses" consistently in the title, wording and check. Not a defined term, but my hope is consistent use within the failure can give clarity.

Reply:
naw i get where you're coming from.  i'd still rather we not use the word 'must' per everything that's been drilled into me about not using such terms in informative docs.  wcag is far looser with must and should than any other doc i've worked on.

Reply:
When in doubt, take it out :)
I took your suggested rewording but removed "might".
If it looks fine now, please mark the conversation as resolved.